source/BoardingPanel.cpp

```diff
@@ -107,10 +109,56 @@ BoardingPanel::BoardingPanel(PlayerInfo &player, const shared_ptr<Ship> &victim)
 			plunder.emplace_back(outfit, count);
 	}
 
-	canCapture = victim->IsCapturable() || player.CaptureOverriden(victim);
-	// Some "ships" do not represent something the player could actually pilot.
+	// Determine the capture requirements of the ship being boarded and the tools
+	// that the boarding ship has to capture it.
+	captureRequirements = victim->CaptureRequirements();
+	set<string> captureNames;
+	for(const auto &it : captureRequirements)
+		captureNames.insert(it.first);
+	captureTools = you->CaptureTools(captureNames);
+
+	// Some "ships" do not represent something the player could actually pilot
+	canCapture = (victim->IsCapturable() || player.CaptureOverriden(victim));
 	if(!canCapture)
-		messages.emplace_back("This is not a ship that you can capture.");
+	{
+		// Or may have once been capturable but have since locked down.
+		if(victim->IsLockedDown())
+		{
+			messages.push_back("This ship is locked down.");
+			messages.push_back("It can't be captured.");
+		}
+		else
+			messages.push_back("This is not a ship that you can capture.");
+	}
+	else if(!captureRequirements.empty())
+	{
+		// Other ships may require tools to capture that the player doesn't have.
+		if(captureTools.empty())
```

Comment:
We dont seem to account for the case where the player has tools but none that work for this ship? Or do they all work?